launch_testing/launch_testing/util/proc_lookup.py

```diff
@@ -68,7 +68,7 @@ def cmd_match_fn(proc):
         elif cmd_args is NO_CMD_ARGS:
             return len(proc.process_details['cmd']) == 1
         else:
-            return cmd_args in proc.process_details['cmd'][1:]
```

Comment:
This was just wrong all along - but ever since we started matching process using the launch.actions themselves (a much better idea) nothing inside launch_testing actually uses this so it wasn't noticed.

I stumbled on this while trying to do some tooling that would match tested processes up to documentation, so I need string -> launch action lookup to work correctly.  The new tests should cover the correct behavior well enough to prevent regression